mmhuman3d/core/visualization/visualize_keypoints3d.py

```diff
@@ -78,7 +78,7 @@ def visualize_kp3d(
         mask (Optional[Union[list, tuple, np.ndarray]], optional):
             mask to mask out the incorrect points. Defaults to None.
         start (int, optional): start frame index. Defaults to 0.
-        end (int, optional): end frame index. Defaults to -1.
+        end (int, optional): end frame index. Defaults to None.
```

Comment:
Need to indicate that the `end` frame index is `exclusive`, and the `end` is defined the same way as torch and numpy.

Reply:
already solved. Our `end` serves the same as `list`, `tensor`, and `numpy` slice.